fix(cost-explorer-routes): remove label & copiable attributes

sulmoJ opened the pull request:
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?


### Description
SSIA

### Things to Talk About

yuda110 reviewed (approved):
LGTM!

WANZARGEN reviewed (approved):
Oh, is the requirement updated not to show query set id?
I wonder why. 😯

sulmoJ commented:
> Oh, is the requirement updated not to show query set id?

I don't know why I removed it. I just removed it according to the design.
I will tell you after checking the reason for the removal.
